pkg/iac/rego/load.go

```diff
@@ -165,15 +165,15 @@ func (s *Scanner) fallbackChecks(compiler *ast.Compiler) {
 			continue
 		}
 
-		s.logger.Error(
+		s.logger.Debug(
 			"Error occurred while parsing. Trying to fallback to embedded check",
```

Comment:
nit:
Can we somehow change the word `Error`?
Even if we show debug log, some users see word `error` and think it's an error.

Reply:
Hmm I agree - I updated it here https://github.com/aquasecurity/trivy/pull/8634/commits/1bcf4c2121a68c1606c0724f92fb76ebb04de9e7

In the next release I want to add support inside of checks metadata to specify the minimum supported trivy version that the particular check would require to run. 

That information will be used in this logic to only load those checks which are supported by that particular trivy version, thereby avoiding us getting into situations like these.

Reply:
Details on the new feature here https://github.com/aquasecurity/trivy/issues/8637

Reply:
> Details on the new feature here https://github.com/aquasecurity/trivy/issues/8637

Looks good  👍